logFilter.js

```diff
@@ -13,11 +13,13 @@ function filterLogsByEnv(logs, env) {
 function filterLogsOnDate(logs, dateFilter) {
   const filteredLogs = []
   const parsedDateFilter = new Date(dateFilter)
-
+  console.log(dateFilter)
```

Comment:
I comment I receive on probably 1 out of every 4 PRs I submit, you forgot a `console.log` 😂